pebble.go

```diff
@@ -116,6 +116,9 @@ func (s *PebbleDHStore) GetMetadata(hvk HashedValueKey) (EncryptedMetadata, erro
 
 	emb, emClose, err := s.db.Get(hvkk.buf)
 	_ = hvkk.Close()
+	if err == pebble.ErrNotFound {
+		return nil, nil
+	}
 	if err != nil {
 		return nil, err
 	}
```

Comment:
```suggestion
```suggestion
	if err != nil {
		if errors.Is(err, pebble.ErrNotFound) {
			return nil, nil
		}
		return nil, err
	}
```
```